src/arrayMethodJoin.js

```diff
@@ -4,8 +4,22 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let string = '';
```

Comment:
```suggestion
    let concatenatedString = '';
```
Bad idea to name variables with the data-type name. It's better to use `joinedString` or `concatenatedString` (or even `result` at least)